backend/test/initialization/initialization_test.go

```diff
@@ -0,0 +1,55 @@
+package initialization
+
+import (
+	"testing"
+
+	"github.com/golang/glog"
+	params "github.com/kubeflow/pipelines/backend/api/go_http_client/experiment_client/experiment_service"
+	"github.com/kubeflow/pipelines/backend/src/common/client/api_server"
+	"github.com/kubeflow/pipelines/backend/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type InitializationTest struct {
+	suite.Suite
+	namespace        string
+	experimentClient *api_server.ExperimentClient
+}
+
+// Check the namespace have ML job installed and ready
+func (s *InitializationTest) SetupTest() {
+	if !*runIntegrationTests {
```

Comment:
What's the purpose of this switch?

BTW, `go test` supports test case regex which allows only running the tests you want.

Reply:
This prevents the test from running as part of the the travis CI tests